qcschema/dev/dev_schema.py

```diff
@@ -10,17 +10,16 @@
 
 # The base schema definition
 base_schema = {
-    "$schema": "http://json-schema.org/draft-06/schema#",
-    "name": "QC_JSON",
-    "version": "0.1.dev",
-    "url": "http://schema_host.org/schemas/0/something.schema",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "name": "qc_schema_input",
+    "version": "dev",
     "description": "The MolSSI Quantum Chemistry Schema",
     "type": "object",
     "properties": {
         "molecule": molecule.molecule,
         "schema_name": {
             "type": "string",
-            "pattern": "\W*(QC_JSON)\W*"
+            "pattern": "\W*(qc_schema)\W*"
```

Comment:
why `qcschema` in docs and dir structure and `qc_schema` elsewhere? I'm glad of the change from `QC_JSON`, both in caps and name.

Reply:
`qcschema` is canonical Python for modules (I dont think this is in the docs?) while `qc_schema` is the keyword snake case.

Reply:
Thanks for the distinction – makes much more sense now. (reference to docs was the sphinxproj name.)